UI.tsv

```diff
@@ -375,7 +375,7 @@ UI_20150317_000374	{@st42b}Price
 UI_20150317_000375	{@st42b}Hide Seller's Name{/}
 UI_20150317_000376	{@st41b}Add
 UI_20150317_000377	Siege Coin
-UI_20150317_000378	Someone killed someone
+UI_20150317_000378	Someone was killed by someone
```

Comment:
This version is better since it's talking in 3rd person. If the player was involved, active voice would be better "You killed someone"